forge-gui/res/cardsfolder/c/caverns_of_despair.txt

```diff
@@ -5,4 +5,5 @@ K:No more than two creatures can attack each combat.
 S:Mode$ Continuous | GlobalRule$ No more than two creatures can block each combat. | Description$ No more than two creatures can block each combat.
 SVar:NonStackingEffect:True
 AI:RemoveDeck:Random
+DeckHints:Type$Planeswalker|Artifact|Enchantment|Aura|Equipment
```

Comment:
I don't see it?

Reply:
seems pretty ok in noncreature decks or aura/equipment decks ?